README.adoc

```diff
@@ -1,56 +1,51 @@
 = FeatureHub Webhook Utility Package
 
 This package is designed to give you utilities which allow you to transform the incoming
-webhook from FeatureHub into "other things". 
+webhook from FeatureHub into a message format, that can be used by one of the FeatureHub SDKs, and to allow to store the message at a destination (e.g. S3 bucket). Primary use case for this is caching features outside FeatureHub server, for example in AWS S3 storage or Cloud Storage.
```

Comment:
the grammar in these two sentences is a bit weird? "a message format, that can be used by any of the FeatureHub SDKs, and allow storing the message at a destination (e.g. S3 bucket). The primary"